osidb/models.py

```diff
@@ -2704,11 +2692,19 @@ def fix_state(self):
             self.resolution = self.resolution.lower()
 
         # Eg. GITOPS-1472, AAH-682
-        if self.status in ("won't fix", "obsolete"):
-            return Affect.AffectFix.WONTFIX
+        if self.status in ("won't fix", "obsolete", "deferred"):
+            # won't fix should display/should handle as as DEFER
+            return Affect.AffectFix.DEFER
         if self.status in ("done", "resolved", "closed"):
-            if self.resolution in ("won't do", "won't fix", "wontfix", "obsolete"):
```

Comment:
This mapping is wrong. We should just take the values from BTS as they were. There is no change in the meaning of the old or BTS values. Still **won't fix** etc. means WONTFIX even though it is historical value now. Additionally, we are changing the value which then go to the CVE pages - the new and the historical. This should be all reverted back

Reply:
There was a request to change the data value to 'DEFER' and that there was not a value to having the historical data. See OSIDB-3289 for more details.

Reply:
This requirement has reverted, WONTFIX is to be supported. Will revert this.